assets/script.js

```diff
@@ -4,26 +4,32 @@ var searchBtn = document.querySelector("#search-button");
 var dropDown = document.querySelector("#drop-down");
 var searchForm = document.querySelector("#search-form");
 var checkBoxContainer = document.querySelector("#checkbox-container");
-var filmCheck =document.querySelector("#film");
+var filmCheck = document.querySelector("#film");
 var tvCheck = document.querySelector("#tv");
-var userInput = document.querySelector("#user-input");
+var userInput = document.querySelector("#user-input").value;
```

Comment:
I think leave the .value off up here because there are parts of the code below that use userInput.value below and that's going to cause some weirdness.